source/switch_payment_service_provider/switch_psp_to_stripe/index.html.md.erb

```diff
@@ -0,0 +1,116 @@
+---
+title: Switch payment service provider to Stripe
+last_reviewed_on: 2021-07-07
+review_in: 6 months
+weight: 167
+---
+
+# Switch payment service provider to Stripe
+
+This guidance tells you how to switch your payment service provider (PSP) to Stripe.
+
+You must be a GOV.UK Pay admin user to switch PSP. To check if you are an admin user for a service:
+
+1. Select **View team members** or **Manage team members** next to the service on the **Overview** page.
+
+2. If your email address is listed under **Administrators**, you are an admin user.
+
+## Prepare to switch to Stripe
+
+Email [govuk-pay-support@digital.cabinet-office.gov.uk](mailto:mailto:govuk-pay-support@digital.cabinet-office.gov.uk) asking to switch your PSP to Stripe on all services for your organisation, or list out specific services.
+
+The GOV.UK Pay support team will confirm when we’ve enabled the switch PSP feature for your service.
+
+To switch to Stripe, you’ll need:
+
+* the name, date of birth, and home address of your organisation’s [‘responsible person’](https://www.payments.service.gov.uk/what-being-a-responsible-person-means/)
+* your bank details
+* your VAT number
+* your company registration number (if your organisation has one)
+* a debit or credit card to make a refundable payment to test your service
+
+Before switching, read the:
+
+* [GOV.UK Pay contract](https://selfservice.payments.service.gov.uk/policy/download/contract-for-non-crown-bodies) (if you are a non-Crown organisation)
+* [Memorandum of Understanding](https://selfservice.payments.service.gov.uk/policy/download/memorandum-of-understanding-for-crown-bodies) (if you are a Crown organisation)
+* [Stripe Connected Account Agreement](https://selfservice.payments.service.gov.uk/policy/download/stripe-connected-account-agreement).
+
+To get to the **Switch payment service provider** page:
+
+1. Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login).
+
+2. Select the service you want to switch from the **Overview** page - services you can switch are labelled **LIVE - SWITCH PSP**
+
+3. On your service dashboard, select the ‘**...how to switch to Worldpay link**’ in the banner at the top of the page, or select **Settings** in the navigation.
+
+4. In **Settings**, select **Switch PSP** from the side navigation.
+
+## Switch to Stripe
+
+You need to repeat the following process for each service you want to switch.
+
+### Provide your details and test the integration with Stripe
+
+To test the integration between Stripe and GOV.UK Pay, you need to make a live payment. This step makes sure you have integrated your GOV.UK Pay and Stripe accounts correctly.
+
+The live payment is £2.00 and is refundable (minus the transaction fee).
```

Comment:
Could this be a tiny bit confusing as to whether the person making their payment only gets back an amount minus the transaction fee, rather than them getting back the full amount but the service's new Stripe account being debited the transaction fee?